test/testutility.cpp

```diff
@@ -67,38 +69,34 @@ private Q_SLOTS:
         QLocale::setDefault(QLocale(QStringLiteral("C")));
         //NOTE: in order for the plural to work we would need to load the english translation
 
-        quint64 sec = 1000;
-        quint64 hour = 3600 * sec;
-
-        QDateTime current = QDateTime::currentDateTimeUtc();
-
-        QCOMPARE(durationToDescriptiveString2(0), QString::fromLatin1("0 second(s)"));
-        QCOMPARE(durationToDescriptiveString2(5), QString::fromLatin1("0 second(s)"));
-        QCOMPARE(durationToDescriptiveString2(1000), QString::fromLatin1("1 second(s)"));
-        QCOMPARE(durationToDescriptiveString2(1005), QString::fromLatin1("1 second(s)"));
-        QCOMPARE(durationToDescriptiveString2(56123), QString::fromLatin1("56 second(s)"));
-        QCOMPARE(durationToDescriptiveString2(90 * sec), QString::fromLatin1("1 minute(s) 30 second(s)"));
-        QCOMPARE(durationToDescriptiveString2(3 * hour), QString::fromLatin1("3 hour(s)"));
-        QCOMPARE(durationToDescriptiveString2(3 * hour + 20 * sec), QString::fromLatin1("3 hour(s)"));
-        QCOMPARE(durationToDescriptiveString2(3 * hour + 70 * sec), QString::fromLatin1("3 hour(s) 1 minute(s)"));
-        QCOMPARE(durationToDescriptiveString2(3 * hour + 100 * sec), QString::fromLatin1("3 hour(s) 2 minute(s)"));
-        QCOMPARE(durationToDescriptiveString2(current.msecsTo(current.addYears(4).addMonths(5).addDays(2).addSecs(23 * 60 * 60))),
+        const QDateTime current = QDateTime::currentDateTimeUtc();
+
```

Comment:
Why aren't we getting the proper plural forms here?
Apparently we never got them.